README.md

```diff
@@ -0,0 +1,31 @@
+imgase-back
+=====
+
+`images-back` is the webservice to store and process images.
+
+Installation
+------------
+
+Install `images-back` from sources, by running:
+
+```sh
+git clone https://github.com/PhilLar/Images-back.git
+cd short
+go install ./cmd/images-back
+```
+
+Usage
+-----
+You can run it:
+```sh
+images-back
```

Comment:
Let's add and example of specifying the database URL:
```suggestion
export DATABASE_URL="postgres://images:secret@localhost/imagesapp?sslmode=disable"
images-back
```

Reply:
About `go mod tidy`: https://github.com/golang/go/wiki/Modules#how-to-prepare-for-a-release